security/medusa/l2/kobject_file.c

```diff
@@ -160,6 +162,36 @@ void file_kobj_dentry2string_dir(struct path *dir, struct dentry *dentry, char *
 	buf[len] = '\0';
 	dput(dentry);
 }
+
+/* Uses mnt from dir and then gets ndupper as original file_kobj_dentry2string() */
+void file_kobj_dentry2string_mnt(struct path *dir, struct dentry * dentry, char * buf)
```

Comment:
zmenil by som `struct path *dir` na `struct vfsmount *mnt`, nakolko sa iba tato polozka z `dir` pouziva

Reply:
This is an older version.